MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -2877,6 +2877,13 @@ public void processNewDayPersonnel() {
                     u.resetPilotAndEntity();
                 }
             }
+            
+            // Reset edge points to the purchased value each week. This should only
+            // apply for support personnel - combat troops reset with each new mm game
+            if ((p.isAdmin() || p.isDoctor() || p.isEngineer() || p.isTech())
```

Comment:
In Person you're checking for isSupport, but not here. Should those two places be consistent?

Reply:
See above about the differences between isSupport and isAdmin/Doctor/Tech.  In part I was experimenting with the different ways to handle this, and in the other part, I didn't want to reset edge for Astechs and Medtechs - I have a lot of Astech/Vehicle Crews and Soldier/Medtechs in my campaigns that I wanted to handle as combat personnel edgewise.    If we need consistency or if I get some compelling reasons from people who play differently I can change it.